sos/report/plugins/openshift_ovn.py

```diff
@@ -27,6 +27,12 @@ def setup(self):
             "/var/log/openvswitch/libreswan.log",
             "/var/log/openvswitch/ovs-monitor-ipsec.log"
         ])
+        # Collect ovn interconnect specific files if exists.
+        self.add_copy_spec([
+            "/var/lib/ovn-ic/etc/ovnnb_db.db",
+            "/var/lib/ovn-ic/etc/ovnsb_db.db",
+            "/var/log/ovn-ic/acl-audit-log.log"
+        ])
```

Comment:
I think `/var/run/ovn` folder also changed

Reply:
yes, but it is still mounted as /var/run/ovn/ into ovnkube-node container, so i guess we're fine there.

Reply:
the problem is sos report runs on the host, not in a container. You can test these commands e.g. if you `chroot /host` from the debug pod

Reply:
I guess we need to add collection for ovnkube-controller container?

Reply:
Yes @npinaeva , the container commands don't work anymore due to directory path mismatch. now collecting ovs `ct-zone-list` directly on the host which is working fine.